pageObject/accountPage.js

```diff
@@ -13,7 +14,7 @@ class AccountPage extends BasePage {
     }
 
     getBaseElement() {
-        return new TextBox(element(userNameLocator), "User name");
+        return new BaseElement(element(userNameLocator), "User name").waitForVisible();
```

Comment:
No need for `.waitForVisible()` it is getter which should return the only element